target/classes/messages.properties

```diff
@@ -0,0 +1,4 @@
+org.entando.error.engine.notFound=Engine not supported
+org.entando.error.engine.notSupported=Engine not supported for this operation
+
+org.entando.error.task.notFound=Task not found
```

Comment:
Add new line at the end of the file.

Reply:
actually this shouldn't be commited, it's build files